### ISSUE TITLE
Performance of indiscriminate attack 

### ISSUE THREAD

**Issue by hyejin-s** [inspire-group/tta_risk#1]
Hello, I have a question regarding the performance of indiscriminate attacks in your research paper. I want to make sure I understand the methodology correctly. In the paper, it appears that indiscriminate attacks aim to decrease the performance of all benign samples. However, in your provided code, it seems that the adversarial accuracy is measured using all samples, which includes both benign and malicious samples.

I have considered using the following measure for indiscriminate attacks: acc_benign_af = (outputs_adv.max(1)[1][:-source] == y[:-source]).float().sum(). However, when I implement this measure, the performance of the indiscriminate attack is not as effective as the results presented in your paper.

Could you please provide some clarification on this issue? I'm trying to reproduce your code, but I'm unsure about the discrepancy between my results and those in the paper. Additionally, could you share the relevant checkpoint file to assist in my replication efforts?

**Comment by tongwu2020** [comment]
Hi, 

Thank you for your question. 
Your understanding of indiscriminate attacks is correct. What performance do you get for the indiscriminate attack? Could you try changing the loss function in  ```generate_attacks``` function to the following:

```
loss = nn.CrossEntropyLoss(reduction='none')(out[:-source], y[:-source]).clamp(min=0,max=5)
loss = loss.sum()
adv.data = (adv + alpha*adv.grad.detach().sign()).clamp(-epsilon,epsilon)
```

For checkpoints, which one do you need to replicate the results? For most experiments, the checkpoints came from RobustBench.

**Comment by hyejin-s** [comment]
Thank you for your prompt response. I had a misunderstanding regarding the meaning of 'if cfg.ATTACK.TARGETED == False' in the code, which I thought indicated an indiscriminate attack.

Regarding the checkpoint file, I noticed that there is no "resnet26" model available in the Robustbench repository. Could you assist me in locating the appropriate checkpoint file?

**Comment by tongwu2020** [comment]
Hi, could you try https://www.dropbox.com/sh/ztylx8x6yjd8gpl/AACbf-DfFxF0hdWXzqDk-tI3a?dl=0 for models

**Comment by hyejin-s** [comment]
Thank you for your help :)
